src/main/java/org/aksw/simba/lemming/RemoveEdgeDecorator.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * 
+ */
+package org.aksw.simba.lemming;
+
+import it.unimi.dsi.fastutil.ints.IntArrayList;
+import it.unimi.dsi.fastutil.ints.IntSet;
+
+/**
+ * 
+ * Concrete Decorator class to simulate removing of an edge to the ColouredGraph
+ * object in the current iteration
```

Comment:
"... to simulate the _removal_ of an edge _from_ the _IColouredGraph_ object."

Reply:
Changed